lib/source/encoding.pm

```diff
@@ -0,0 +1,72 @@
+package source::encoding;
+
+use strict;
+use warnings;
```

Comment:
Why not `use v5.40;`?

Reply:
I don't see that v5.40 includes warnings.  Where is that documented?

Reply:
https://perldoc.perl.org/perl5360delta#use-v5.36

> Furthermore, `use v5.36` will also enable warnings as if you'd written `use warnings`.

https://perldoc.perl.org/functions/use#use-VERSION

> If the specified Perl version is 5.35.0 or higher, [warnings](https://perldoc.perl.org/warnings) are enabled.

Reply:
changed